internal/dataBase/dataBase.go

```diff
@@ -0,0 +1,160 @@
+package dataBase
+
+import (
+	"database/sql"
+	"log"
+	"os"
+	"path/filepath"
+	"strconv"
+
+	"github.com/azoma13/go_final_project/configs"
+)
+
+var DB *sql.DB
+
+func DbFunc() {
+
+	appPath, err := os.Executable()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	dbFile := filepath.Join(filepath.Dir(appPath), "scheduler.db")
+
+	_, err = os.Stat(dbFile)
+
+	var install bool
+	if err != nil {
+		install = true
+	}
+
+	DB, err := sql.Open("sqlite", "scheduler.db")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer DB.Close()
+	if install {
+		_, err = DB.Exec(`CREATE TABLE IF NOT EXISTS scheduler (
+			id INTEGER PRIMARY KEY AUTOINCREMENT,
+			date CHAR(8) NOT NULL,
+			title TEXT NOT NULL,
+			comment TEXT,
+			repeat VARCHAR(128) NOT NULL
+			);`)
+		if err != nil {
+			log.Fatal(err)
+		}
+		_, err = DB.Exec(`CREATE INDEX IF NOT EXISTS scheduler_date ON scheduler (date);`)
+		if err != nil {
+			log.Fatal(err)
+		}
+	}
+}
+
+func AddTask(date, title, comment, repeat string) (string, error) {
+
+	db, err := sql.Open("sqlite", "scheduler.db")
+	if err != nil {
+		return "", err
+	}
+	defer db.Close()
+
+	result, err := db.Exec("INSERT INTO scheduler (date, title, comment, repeat) VALUES (?, ?, ?, ?)", date, title, comment, repeat)
+	if err != nil {
+		return "", err
+	}
+
+	id, err := result.LastInsertId()
+	if err != nil {
+		return "", err
```

Comment:
Лучше все ошибки оборачивать, чтобы сразу понимать где они произошли
```suggestion
		return "", fmt.Errorf("error while getting lasi inserted id: %w", err)
```